spells/unlock_player.js

```diff
@@ -0,0 +1,36 @@
+// Unlocks a player that is targeted with a "locked spell" fx. annoy
+// Takes two params, context = object containing functions to operate on state,
+// payload = data sent from client-side ex username or player-id for the targeted player
+function unlockPlayer(context, payload) {
+  // First we'll get the game state
+  const game = context.getGameState();
+
+  // In this spell we'll receive the player id (from frontend) (the player to be unlocked)
+  const playerId = payload.player;
```

Comment:
Since the payload is `playerId` this should be `const playerId = payload.playerId;` (https://github.com/VilmaNilsson/VIP2021-game/issues/26)